readme.md

```diff
@@ -61,6 +61,14 @@
 - [Dojo Toolkit](https://github.com/peterkokot/awesome-dojo)
 - [Inspiration](https://github.com/NoahBuscher/Inspire)
 
+## Web development
+
+- [Django](https://github.com/rosarior/awesome-django)
+- [Play1 Framework](https://github.com/PerfectCarl/awesome-play1)
+- [CakePHP](https://github.com/dereuromark/awesome-cakephp)
+- [Symfony2](https://github.com/EmanueleMinotto/awesome-symfony2)
+- [Laravel](https://github.com/tuwannu/awesome-laravel)
+- [Learn to Program](https://github.com/karlhorky/learn-to-program)
```

Comment:
I don't see why you would limit yourself to just Web Development. Learning to program is about a lot more than that. Also from looking at that list a more proper title would be `Back-end developement`, but that should be a separate PR.

So, update this PR to only add your entry to the misc category and do a separate one for the back-end category ;)